py/server/deephaven/table.py

```diff
@@ -1389,6 +1390,30 @@ def partition_by(self, by: Union[str, Sequence[str]], drop_keys: bool = False) -
         except Exception as e:
             raise DHError(e, "failed to create a partitioned table.") from e
 
+    def update_by(self, ops: Union[UpdateByClause, List[UpdateByClause]],
```

Comment:
I doubt you meant to include all of this update by stuff in the datetime numpy fix.

Reply:
absolutely right